modules/release-notes/con-relnotes-known-issues.adoc

```diff
@@ -3,36 +3,18 @@
 
 This section lists known issues with {product} {product-version}:
 
-Helm Chart upgrade release failing on Openshift Developer Console::
-+
---
-If you have installed {product-short} manually using the Helm CLI, the helm chart release upgrade fails in the OpenShift Developer Console. 
+link:https://access.redhat.com/security/cve/CVE-2024-35195[CVE-2024-35195]::
```

Comment:
if these CVEs are FIXED then they are no longer KNOWN ISSUES, but rather SECURITY FIXES in this release.